src/services/cli-http-client.js

```diff
@@ -140,6 +137,20 @@ class CliRequestClient {
     this.logger.debug(`User-Agent: ${options.headers['User-Agent']}`);
     this.logger.debug('-- END Twilio API Request --');
   }
+
+  /* eslint-disable camelcase */
+  // In the rare event parameters are missing, display a readable message
+  formatErrorMessage({ code, message, more_info, details }) {
+    const moreInfoMessage = more_info ? ` See ${more_info} for more info.` : '';
```

Comment:
nit: remove space before `See`

Reply:
Left the space in the front there since the string it's added to ends in `.`. I can add the space to the end of that string instead and remove the space here.